plugins/code-coverage/src/components/FileExplorer/CoverageRow.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { FC } from 'react';
+
+// styles
+import { makeStyles } from '@material-ui/core';
+import { red, green } from '@material-ui/core/colors';
+
+const useStyles = makeStyles(theme => ({
+  lineNumberCell: {
+    color: `${theme.palette.grey[500]}`,
+    fontSize: '90%',
+    borderRight: `1px solid ${theme.palette.grey[500]}`,
+    paddingRight: '8px',
+    textAlign: 'right',
+  },
+  hitCountCell: {
+    width: '50px',
+    borderRight: `1px solid ${theme.palette.grey[500]}`,
+    textAlign: 'center',
+    color: 'white', // need to enforce this color since it needs to stand out against colored background
+    paddingLeft: `${theme.spacing(1)}`,
```

Comment:
```suggestion
    paddingLeft: theme.spacing(1),
```

this helper is made to be useable directly
(and same below)